libs/io/bbhnet/io/h5.py

```diff
@@ -0,0 +1,55 @@
+from typing import TYPE_CHECKING, Optional, Tuple
+
+import h5py
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+import numpy as np

```

Comment:
This should be under the TYPE_CHECKING `if` statement, correct?

Reply:
Done